### PR TITLE
:art: Cleanup old/ directory

### PR DESCRIPTION
**Pull request by wesen**

### CODE REVIEW

File: include/MFixedPoint/FpS.hpp [hunk 1]
```diff
@@ -34,398 +34,272 @@ namespace MFixedPoint {
...
+    //===============================================================================================//
+
+    /// \brief		Create a fixed-point value from a integer and a num. of fractional bits.
+    FpS(int32_t integer) {
+        static_assert(std::is_integral<BaseType>::value, "Integral BaseType required for FpS class.");
+        rawVal_ = integer << numFracBits_;
+    }
+
+    /// \brief		Create a fixed-point value from a double and a num. of fractional bits.
+    FpS(double dbl) {
```
Comment (robweiss) [inline include/MFixedPoint/FpS.hpp]: I thought you wanted to get rid of the double stuff?

Comment (wesen) [reply include/MFixedPoint/FpS.hpp]: I think I kept it to make things like `1.3__fp__32` work. I guess I just didn't compile the unit tests anymore :)

File: test/FpF/FpFComparisonOperators.cpp [hunk 2]
```diff
@@ -0,0 +1,89 @@
+//!
```
Comment (robweiss) [inline test/FpF/FpFComparisonOperators.cpp]: It would be worthwhile adding some comparison tests that show large and extremely large decimal values

Comment (wesen) [reply test/FpF/FpFComparisonOperators.cpp]: this preq looks all weird to me, let me close and reopen.

**Pull request merged**